src/components/Title/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Text, View, StyleSheet} from 'react-native';
+
+Title = ({style,value}) => {
+  return (
+    <View
+      style={styles.container}>
+      <Text style={[styles.textStyle,style]}>{value}</Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Try to always put styles in separate files. Even if it is a single style. It helps us in segregation.